addons/mail/models/res_partner.py

```diff
@@ -232,7 +232,7 @@ def _to_store(self, store: Store, /, *, fields=None, main_user_by_partner=None):
                 data["country"] = {"code": c.code, "id": c.id, "name": c.name} if c else False
             if "display_name" in fields:
                 data["displayName"] = partner.display_name
-            if 'user' in fields:
+            if "user" in fields or "signature" in fields:
```

Comment:
I would not add `or` here, if `user` is not provided `signature` shouldn't be provided either/make no sense.